app/templates/home.html

```diff
@@ -1 +1,6 @@
-<h1>Home Page</h1>
\ No newline at end of file
+<section class="chatroom-list-bar">
+    <h1 id="chatroom-list-title">Bloc Chat</h1>
+    <ul class="chatroom-list-container" ng-repeat="rooms in home.chatRooms.all">
+        <li><a class="chatroom-list-name" ui-sref="{{ rooms }}">{{ "Room " + rooms.$id }}</a></li>
```

Comment:
I would consider using a name from firebase, instead of hardcoding "Room " + id as the format for all chat rooms.  What if you want one called "My super cool chat room"?